docs/specifications/tests/Basic-TP/basic01.md

```diff
@@ -1,52 +1,198 @@
 ## BASIC01: The domain must have a parent domain
 
 ### Test case identifier
-**BASIC01** The domain must have a parent domain
+**BASIC01**
 
 ### Objective
 
-In order for a domain to work, it must be delegated to from a domain higher
-up in the DNS hierarchy (a parent domain). However, note that this
-requirement does not concern the root domain.
+In order for a domain (zone) to work, it must be delegated from a 
+zone higher up in the DNS hierarchy (a parent domain/zone). 
+This Test Case will determine if parent and child zones exist.
+
+If the zone to be tested is the root zone, it has no parent or
+delegation and will always pass this Test Case.
+
+If no parent can be determined, there cannot be any delegation.
+
+If the child zone does not exist (is not delegated), the only 
+test case to be run after this test case is BASIC03. However, 
```

Comment:
Basic03 can be run even if the child zone does not exist. From [Basic03]:

> The case where the delegation for a domain is too broken to be fully tested but functional enough for simple web browsing should be detected. This test should only be performed if the BASIC02 test has failed, in order to explain why the domain seemingly works but otherwise is untestable.

Yes, if there is no delegation, the child zone does not exist.



[Basic03]: https://github.com/zonemaster/zonemaster/blob/develop/docs/specifications/tests/Basic-TP/basic03.md